cli/generic/forms/fields.py

```diff
@@ -100,6 +101,24 @@ def _deserialize(self, value, attr, obj, **kwargs):
         return value
 
 
+class BatchIdentifierField(fields.Field):
+    """
+    Implements validation of the identifier.
+
+    References:
+        - https://marshmallow.readthedocs.io/en/3.0/custom_fields.html
+    """
+
+    def _deserialize(self, value, attr, obj, **kwargs):
+        """
+        Validate data (identifier) that was passed to field.
```

Comment:
`identifier` > `batch identifier`.

Reply:
Add reassigning to the value:

```python
batch_identifier = value
```

And use `value` then. Some functionality in the project does not follow this, but it is pushed to the technical debt to be fixed soon.

Reply:
Done.